sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -163,7 +164,7 @@ export class StudentService extends RecordDataModelService<Student> {
     }
 
     student.user = user;
-    student.birthDate = new Date(userInfo.birthdate);
+    student.birthDate = getISODateOnlyString(new Date(userInfo.birthdate));
```

Comment:
The `getISODateOnlyString` accepts either a Date or string, conversions are not needed.

Reply:
I believe I can directly update like this without any conversion `student.birthDate = userInfo.birthdate;` as `userInfo.birthdate` is the token birthdate value.